portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -1014,6 +1014,273 @@ definition {
 			fieldValue = "1/1/16");
 	}
 
+	@description = "LPS-187219 - Verify that the user can download a file"
+	@priority = 4
+	test CanDownloadFile {
```

Comment:
Hey @alanafergs, since this test has more requirements than simply downloading a file, we might want to make this more specific and describe that we are downloading a file of a rejected form entry via My Workflow Tasks. Please make sure both the title and description describes that. Let me know if you need help with summarizing or wording them!